utils/dependency_management.py

```diff
@@ -379,6 +379,9 @@ def check_requirements(extras, github_annotate):  # pylint disable: too-many-loc
             for fn_req in sorted(fn_reqs):
                 error_msg.append(f' - {fn_req.relative_to(ROOT)}')
 
+        error_msg.append('')
+        error_msg.append('To update the files, you can use: `pip install tox; tox -e py{36,37,38}-update-reqs`')
```

Comment:
I just tested this and overall it seems to work fine as long as you have all the Python versions installed.

However, maybe I was bit naive about this, but I literally executed the command `pip install tox; tox -e py{36,37,38}-update-reqs` which meant that only the Python 3.6 requirements were updated and the command did not fail otherwise. I assume that's not what is meant here? Is there a way to just update all requirements at once? The reason that I expected that is because usually tox runs all the various tests with only one command. If that's not possible, that's also fine, but maybe we should slightly revise that error message hint.